File: 20211001-fibonacci-java/src/test/java/com/katas/FibonacciSequenceTest.java
package com.katas;

import org.junit.Test;

import static org.assertj.core.api.AssertionsForClassTypes.assertThat;

public class FibonacciSequenceTest {

    private final FibonacciSequence fibonacciSequence = new FibonacciSequence();

    @Test
    public void first_number_in_sequence_is_zero() {
        assertThat(fibonacciSequence.numberAt(1)).isZero();
    }

    @Test
    public void second_number_in_sequence_is_one() {
        assertThat(fibonacciSequence.numberAt(2)).isOne();
    }

    @Test
    public void third_number_in_sequence_is_one() {
        assertThat(fibonacciSequence.numberAt(3)).isOne();
    }

    @Test
    public void fourth_number_in_sequence_is_two() {
        assertThat(fibonacciSequence.numberAt(4)).isEqualTo(2);
    }

    @Test
    public void sixth_number_in_sequence_is_five() {
        assertThat(fibonacciSequence.numberAt(6)).isEqualTo(5);
    }
}

File: 20200614-kotlin-fibonacci/src/test/kotlin/com/katas/FibonacciSequenceTest.kt
package com.katas

import org.assertj.core.api.Assertions.assertThat
import org.junit.jupiter.api.Test

class FibonacciSequenceTest {

    private val fibonacciSequence = FibonacciSequence()

    @Test
    internal fun `first number in sequence is 0`() {
        assertThat(fibonacciSequence.numberAt(0)).isZero()
    }

    @Test
    internal fun `second number in sequence is 1`() {
        assertThat(fibonacciSequence.numberAt(1)).isEqualTo(1)
    }

    @Test
    internal fun `third number in sequence is 1`() {
        assertThat(fibonacciSequence.numberAt(2)).isEqualTo(1)
    }

    @Test
    internal fun `fourth number in sequence is 2`() {
        assertThat(fibonacciSequence.numberAt(3)).isEqualTo(2)
    }

    @Test
    internal fun `sixth number in sequence is 5`() {
        assertThat(fibonacciSequence.numberAt(5)).isEqualTo(5)
    }
}